web/templates/compose.hbs

```diff
@@ -60,7 +60,7 @@
                             </div>
                             <div id="compose_recipient_selection_dropdown" class="new-style" tabindex="0">
                                 <div class="stream_header_colorblock"></div>
-                                <button id="compose_select_recipient_widget" class="dropdown-widget-button" type="button">
+                                <button id="compose_select_recipient_widget" class="dropdown-widget-button" type="button" tabindex="-1">
```

Comment:
Hmm, does this make it not work to `Shift+Tab` to the input? Oh, probably it's fine, since we have an explicit keyboard handler for that action.

Reply:
(But how does this part help? I feel like I'm missing something)

Reply:
It doesn't make different since we are handling everything via JS but this is to make sure we don't fallback to `button` on focus but rather use `compose_recipient_selection_dropdown`